client/src/proxy.conf.json

```diff
@@ -0,0 +1,6 @@
+{
+  "/calc_damage": {
+    "target": "http://localhost:5000",
```

Comment:
commenting so i don't forget this but is it possible to set this up so that when running locally it sends requests to localhost, but in prod it sends requests to the actual backend?